app/views/conferences/_form.html.slim

```diff
@@ -16,6 +16,7 @@
   = f.input :flights
   = f.input :round
   = f.input :accomodation
+  = f.input :lightningtalkslots, input_html: {value: nil}, label: "Lightning Talk Slots"
```

Comment:
The `value: nil` is still here. See my previous comment on that line for the reasoning behind it.